modules/server/src/test/scala/server/RpcServerTestSuite.scala

```diff
@@ -64,31 +63,36 @@ trait RpcServerTestSuite extends RpcBaseTestSuite {
 
   object implicits extends Helpers with DummyData {
 
-    def idApply[A](fa: GrpcServer.Op[A]): Id[A] = {
-      import GrpcServer._
-      fa match {
-        case StartOp()                       => serverMock.start()
-        case GetPortOp()                     => serverMock.getPort
-        case GetServicesOp()                 => serverMock.getServices.asScala.toList
-        case GetImmutableServicesOp()        => serverMock.getImmutableServices.asScala.toList
-        case GetMutableServicesOp()          => serverMock.getMutableServices.asScala.toList
-        case ShutdownOp()                    => serverMock.shutdown()
-        case ShutdownNowOp()                 => serverMock.shutdownNow()
-        case IsShutdownOp()                  => serverMock.isShutdown
-        case IsTerminatedOp()                => serverMock.isTerminated
-        case AwaitTerminationTimeoutOp(t, u) => serverMock.awaitTermination(t, u)
-        case AwaitTerminationOp()            => serverMock.awaitTermination()
-      }
-    }
+    def grpcServerHandlerTests[F[_]](implicit F: Applicative[F]): GrpcServer[F] = {
```

Comment:
`F[_]: Applicative`

Reply:
I preferred this way in order to use `F` as `F.pure`, rather than `Applicative[F].pure`.